projects/scion/microfrontend-platform/src/lib/host/application-config.ts

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
Copyright header is missing

Reply:
Fixed indentation in all copyright headers